pkg/controller/controller.go

```diff
@@ -410,6 +411,7 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) {
 
 	glog.V(1).Info("Starting machine-controller-manager")
 	handlers.UpdateHealth(true)
+	prometheus.MustRegister(c)
```

Comment:
It would also be really nice to have a very small docu, just mentioning Prometheus and current-metrics exposed.We plan to have a doku-run soon and complete many of it, can be taken care there.

Reply:
Added a little explanation here.